src/pages/home/report/ReportDetailsExportPage.tsx

```diff
@@ -0,0 +1,135 @@
+
+type ExportType = ValueOf<typeof CONST.REPORT.EXPORT_OPTIONS>;
+
+type ExportSelectorType = SelectorType<ExportType>;
+
+function ReportDetailsExportPage({route}: ReportDetailsExportPageProps) {
+    const connectionName = route?.params?.connectionName;
+    const reportID = route.params.reportID;
+    const [report] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`);
+    const [reportActions] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`);
+    const policyID = report?.policyID;
+
+    const {translate} = useLocalize();
+    const [modalStatus, setModalStatus] = useState<ExportType | null>(null);
+
+    const iconToDisplay = ReportUtils.getIntegrationIcon(connectionName);
+    const canBeExported = ReportUtils.canBeExported(report);
+    const isExported = ReportUtils.isExported(reportActions);
+
+    const confirmExport = useCallback(
+        (type = modalStatus) => {
+            if (type === CONST.REPORT.EXPORT_OPTIONS.EXPORT_TO_INTEGRATION) {
+                ReportActions.exportToIntegration(reportID, connectionName);
+            } else if (type === CONST.REPORT.EXPORT_OPTIONS.MARK_AS_EXPORTED) {
+                ReportActions.markAsManuallyExported(reportID);
+            }
+            setModalStatus(null);
+            Navigation.dismissModal();
+        },
+        [connectionName, modalStatus, reportID],
+    );
+
+    const exportSelectorOptions: ExportSelectorType[] = [
+        {
+            value: CONST.REPORT.EXPORT_OPTIONS.EXPORT_TO_INTEGRATION,
+            text: translate('workspace.common.exportIntegrationSelected', connectionName),
+            icons: [
+                {
+                    source: iconToDisplay ?? '',
+                    type: CONST.ICON_TYPE_AVATAR,
+                },
+            ],
+            isDisabled: !canBeExported,
+        },
+        {
+            value: CONST.REPORT.EXPORT_OPTIONS.MARK_AS_EXPORTED,
+            text: translate('workspace.common.markAsExported'),
+            icons: [
+                {
+                    source: iconToDisplay ?? '',
+                    type: CONST.ICON_TYPE_AVATAR,
+                },
+            ],
+            isDisabled: !canBeExported,
+        },
+    ];
+
+    if (!canBeExported) {
+        return (
+            <ScreenWrapper testID={ReportDetailsExportPage.displayName}>
+                <HeaderWithBackButton title={translate('common.export')} />
+                <ConfirmationPage
+                    illustration={Illustrations.LaptopwithSecondScreenandHourglass}
+                    heading={translate('workspace.export.notReadyHeading')}
+                    description={translate('workspace.export.notReadyDescription')}
+                    shouldShowButton
+                    buttonText={translate('common.buttonConfirm')}
+                    onButtonPress={Navigation.goBack}
+                    illustrationStyle={{width: 233, height: 162}}
+                />
```

Comment:
This is missing flex styles to have the component occupies the whole page. Coming from https://github.com/Expensify/App/issues/58342

Reply:
Ah my bad! You are right!